### DISCUSSION TITLE
Feature Request: Show issue labels on board layout in Projects (beta)

### DISCUSSION THREAD

**Discussion by markdorison** [community/community#5034] [Projects and Issues] [answered]
Issues viewed on existing (non-beta) projects in the "board" layout show labels. Some form of this functionality would be fantastic in the new Projects (beta).

<img width="247" alt="CleanShot 2021-08-11 at 10 48 28@2x" src="https://user-images.githubusercontent.com/20355/129051289-ae9a215b-8b5e-4c49-91b4-1f96b51f6888.png">

**Comment by wulfland** [comment, marked as answer]
This works for me. You have to add the label column in table mode. Then switch back to board and the labels are visible.

**Reply by markdorison** [reply]
Thanks for pointing this out! I already had the labels column in table mode, but I was able to add it to the board view manually.
